packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -383,6 +398,9 @@ LARGE_SPECS = {
     for spec in LARGE_SPECS
 ]
 
+# TODO: test on multiple node versions
+# Some tests use specific ports and can not be run concurrently
+# Some tests fail for unknown reasons when run on multiple platforms
```

Comment:
These tests are probably the most important that should be ran on multiple platforms.

Do you have examples of the failures? 

Probably we can investigate this in the near future.

Reply:
That shouldn’t be hard to do as the CLI does support port 0. 

I can take a look at these maybe in a week.

Reply:
I am happy to get the setup in before that. Unless @clydin has any reservations la

Reply:
I'm going to work on this and push to this PR but you can merge the first commit now if you'd like